queries/just/highlights.scm

```diff
@@ -0,0 +1,102 @@
+
+(comment) @comment
+(string) @string
+(boolean ["true" "false"]) @boolean
+["if" "else"] @conditional
+["export" "alias" "set"] @keyword
+["@" "==" "!=" "+" ":="] @operator
+[ "(" ")" "[" "]" "{{" "}}" "{" "}"] @punctuation.bracket
+
+[
+  (assignment (NAME))
+  (alias (NAME))
+  (value (NAME))
+  (parameter (NAME))
+] @variable
```

Comment:
Are you sure you want to capture the entire assignment with variable, or just it's name?

Reply:
Oh yeah, that was a brain fart. I guess I have to capture each of those individually, huh?